source/testnet/guides/get-started.rst

```diff
@@ -0,0 +1,130 @@
+a six-digit passcode. If you would rather use a full password including letters, you can also choose to do so here.
+
+.. image:: images/concordium-id/int1.png
+      :width: 32%
+.. image:: images/concordium-id/int2.png
+      :width: 32%
+
+.. todo::
+
+   Write a directive to make two or more images side-by-side centered
+
+
+Having chosen either a passcode or a full password, you will get the option to also use biometrics if your phone
+supports it. This can e.g. be facial recognition or fingerprint. We recommend using biometrics if you have the option to do so.
+
+.. image:: images/concordium-id/int3.png
+      :width: 32%
+      :align: center
+
+Request your initial account and identity
+=========================================
+
+Next up, you will get a choice between making a new initial account and identity, or importing an already existing set.
+Assuming this is the first time you are using Concordium ID, you can choose **I want to create my initial account** to continue.
+
+.. image:: images/concordium-id/int4.png
+      :width: 32%
+      :align: center
+
+
+On the next screen you will see a description of what the initial account is and the three steps you have to finish to get it,
+along with your identity. In short, the initial account is an account submitted to the chain by the identity provider of your
+choice, which means they will know that you are the owner of the account. Later you will be able to submit accounts to the
+chain by yourself, which means the owner of these accounts will be only known by you.
+
+.. image:: images/concordium-id/int5.png
+      :width: 32%
+      :align: center
+
+The three steps mentioned above is:
+
+1. Naming of your initial account
+2. Naming your identity
+3. Requesting the initial account and identity from an :ref:`glossary-identity-provider` of your choice
+
+You will meet the first step on the next page, which prompts you to enter a name for your initial account. Pressing continue
+will take you to the next page, on which you have to name your identity. Both of these names will only be known by yourself,
+so you can name them more or less whatever you want (There are a few constraints on what letters and signs you can use).
+
+In the example below, we choose to call our initial account *Example Account 1* and our identity *Example Identity*. As
+mentioned, you can choose whichever names you want.
+
+.. image:: images/concordium-id/int6.png
+      :width: 32%
+.. image:: images/concordium-id/int7.png
+      :width: 32%
+
+By pressing **Continue to identity providers**, you will be taken to a page where you have to choose between *identity providers*.
+An identity provider is an external entity who will verify who you are, before returning an identity object to be used on the chain.
+For now you can choose between:
+
+* *Notabene Development* which will give you a test identity without formal verification.
```

Comment:
I am not sure that "formal" belongs there. I think we should stick to "identity verification" or "real life identity verification".

Reply:
Changed the wording to use "real life identity verification" instead.